plugins/email_service/app/controllers/email_service/application_controller.rb

```diff
@@ -2,11 +2,55 @@
 
 module EmailService
   class ApplicationController < DashboardController
-    authorization_context 'email_service'
-    authorization_required
+    include AwsSesHelper
+    include EmailHelper
+    include TemplateHelper
+    include VerificationHelper
+    include ConfigsetHelper
 
-    def index
-      enforce_permissions('email_service:application_list')
+    def ui_switcher
+      if current_user.has_role?('cloud_support_tools_viewer')
+        redirect_to emails_path
+      end
     end
+ 
+    def restrict_access
+      unless current_user.has_role?('cloud_support_tools_viewer')
+        redirect_to index_path
+      end
+    end 
+
+    def render_paginatable_for_configset(items, filter={})
```

Comment:
Why do you have this method globally when it is just for the configset. Why not in the configset controller? You don't have to changed it is just best practices.

Reply:
done.